scripts/install-elastio.sh

```diff
@@ -0,0 +1,229 @@
+    echo "All commandline arguments are optional."
+    echo "Usage examples:"
+    echo "   $me"
+    echo "   $me -u"
+    echo "   $me --cli-only"
+    echo "   $me --driver-only"
+    echo
+    echo "  -c | --cli-only       : Install Elastio CLI only, without changeds-tracking driver Elastio-Snap."
+    echo
+    echo "  -d | --driver-only    : Install changeds-tracking driver Elastio-Snap only, without Elastio CLI."
+    echo "                          NOTE: There is no Elastio-Snap packages for Fedora (kernel 5.8 and newer) for now."
+    echo "                                So, temporarily this option does nothing on Fedora."
+    echo
+    echo "  -u | --uninstall      : Uninstall all Elastio packages."
+    echo
+    echo "  -h | --help           : Show this usage help."
+}
+
+while [ "$1" != "" ]; do
+    case $1 in
+        -c | --cli-only)        cli=1 ;;
+        -d | --driver-only)     driver=1 ;;
+        -u | --uninstall)       uninstall=1 ;;
+        -f | --force)           force=1 ;;
+        -h | --help)            usage && exit ;;
+        *)                      echo "Wrong arguments!"
+                                usage && exit 15 ;;
+    esac
+        shift
+done
+
+if [ "$EUID" -ne 0 ]; then
+	echo "Run as sudo or root."
+	exit 1
+fi
+
+set -e
+
+if [ ! -z "$uninstall" ]; then
+    uninstall_all
+    echo
+    echo "All elastio packags has been uninstalled!"
+    exit
+fi
+
+# Detect distro name and version
+if [ -f /etc/os-release ]; then
+    dist_name=$(grep ^ID= /etc/os-release | cut -d= -f2 | tr -d '"')
+    dist_ver=$(grep VERSION_ID /etc/os-release | tr -cd '[0-9].' | cut -d'.' -f1)
+else
+    dist_name=$(cat /etc/system-release 2>/dev/null | cut -d' ' -f1)
+    dist_ver=$(cat /etc/system-release 2>/dev/null | tr -cd '[0-9.]' | cut -d'.' -f1)
+fi
+
+# Lowercase dist_name
+dist_name=$(echo ${dist_name,,})
+
+if [ -z "$cli" ] && [ -z "$driver" ]; then
+    cli=1
+    driver=1
+fi
+
+case ${dist_name} in
+    amazon | amzn )
+        if [ $dist_ver -ne 2 ]; then
+            echo "The Amazon Linux 2 is only supported. Current Amazon Linux $dist_ver isn't supported."
+            exit 1
+        fi
+        cent7_amazon_install Amazon $(rpm -E %amzn) amzn
+    ;;
+
+    scientific | sl | oracle | ol )
+        echo "Warning: Oracle Linix and Scientific Linux are not officially supported!"
+        if [ -z $force ]; then
+            echo "We can try to install packages for CentOS on your system."
+            echo "Add '--force' to insist on the installation. But keep in mind, you act at your own risk!"
+            exit 1
+        fi
+    ;;&
+
+    centos | el | rhel | red | scientific | sl | oracle | ol )
+        case ${dist_ver} in
+            7 | 8 ) cent_fedora_install CentOS $(rpm -E %rhel) el ;;
+            * )
+                echo "CentOS/RHEL and derivatives like Oracle Linux and Scientific Linux 7 and 8 are supported. Current distro version $dist_ver isn't supported."
```

Comment:
You just said above that Oracle and SL are not officially supported.

```suggestion
                echo "Old versions of CentOS/RHEL and derivatives like Oracle Linux and Scientific Linux 7 and 8 are not supported. This includes your current distro version $dist_ver."
```

Reply:
This message is about versions out of supported range like CentOS 5, 6, 9 ))
It'll be shown for them. The suggested phrase is confusing, especially "7 and 8 are *not* supported"

Reply:
Let's say shortly just "CentOS/RHEL versions 7 and 8 are supported. Current distro version $dist_ver isn't supported."

Reply:
Using this phrase and re-requesting review.